.ci/no-exception-test.sh

```diff
@@ -60,7 +60,7 @@ openjdk14-with-checks-nonjavadoc-error)
   cd .ci-temp/contribution/checkstyle-tester
   cp ../../../.ci/openjdk-projects-to-test-on.config openjdk-projects-to-test-on.config
   sed -i '/  <!-- Filters -->/r ../../../.ci/openjdk14-excluded.files' checks-nonjavadoc-error.xml
-  export MAVEN_OPTS="-Xmx2048m"
+  export MAVEN_OPTS="-Xmx6144m"
```

Comment:
ANTLR4 caches DFA states to improve performance; when parsing a gigantic repo (50,000 files) like openjdk14, this causes increased memory usage.